scss/styles.scss

```diff
@@ -25,28 +26,36 @@
 }
 
 .main-section {
-  margin: 0 140px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  flex-direction: column;
+  max-width: 1040px;
+  margin: auto;
 }
 
 .technology-group {
-	margin: 100px 0 40px 0;
+  &:not(:first-child){
+    margin: 100px 0 40px ;
```

Comment:
Only `margin-top: 100px;`.
Because if you have 3 containers, the margin between the second and third will be 140px (100 + 40)